test/runtests.jl

```diff
@@ -27,7 +34,7 @@ strip_summary(content::String) = join(split(content, "\n")[2:end], "\n")
 @testset "ReferenceTests" begin
 
 # load/create some example images
-lena = testimage("lena_color_256")
+monarch = testimage("monarch_color_256")
```

Comment:
thank you for this.
It is unrelated to this PR but is an appreciated change.
(For anyone who doesn't know Lena has requested that people stop using her image. https://pursuit.unimelb.edu.au/articles/it-s-time-to-retire-lena-from-computer-science )

Reply:
Moved to https://github.com/JuliaTesting/ReferenceTests.jl/pull/118.